tests/stress_tests/memleaks_tests/tests_pipelines/tests_pipelines.cpp

```diff
@@ -92,14 +92,35 @@ TestResult common_test_pipeline(const std::function<void()>& test_pipeline, cons
 
             // compute test info
             if (iteration >= WARMUP_STEPS) {
+                if (ref != std::array<long, MeasureValueMax> {-1}) {
```

Comment:
example of fail: https://openvino-ci.intel.com/job/private-ci/job/ie/job/stress-tests-linux/349/artifact/b/logs/stress_memleaks_desktop_nightly/StressMemLeaksTests-logs/gtest-parallel-logs/failed/StressMemLeaksTests-MemLeaksTests_MemLeaksTestSuiteNoModel_load_unload_plugin_Numprocesses_1_Numthreads_1_Numiters_1000_Device_GPU_Model_NULLr5nmuupa-1.log